docs/src/guides/run-genbank-ingest-on-terra.rst

```diff
@@ -0,0 +1,65 @@
+*********************************************
+Ingest SARS-CoV-2 data from GenBank on Terra
+*********************************************
+
+We have provided two pipelines for importing data into Terra:
+
+* **GenBank Ingest - pull a public dataset and send them through our preprocessing scripts.**
+* GISAID Ingest - pull a private dataset if a user has their own API endpoint, username, and password.
+
+The pipelines were mainly motivated to provide access to our data pre-processing scripts. Currently, these are focused on pulling datasets for the ncov workflow and is roughly diagrammed below:
+
+.. image:: ../images/terra-ingest.png
+
+This guide describes the **GenBank Ingest**.
+
+Import the GenBank ingest wdl workflow from Dockstore
+======================================================
+
+1. `Set up a Terra account <https://terra.bio/>`_.
+2. Navigate to one of the following in Dockstore:
+    - `nextstrain/ncov/genbank_ingest`_: for open (GenBank) data
+3. At the top right corner, under **Launch with**, click on **Terra**. You may be prompted to log in.
+4. Provide a **Workflow Name** (e.g. ``genbank_ingest`` ).
+5. Select a **Destination Workspace** from the dropdown menu.
+6. Click **IMPORT**.
+7. In your workspace, click on the **WORKFLOWS** tab and verify that the imported workflow is showing a card.
+
+.. _`nextstrain/ncov/genbank_ingest`: https://dockstore.org/workflows/github.com/nextstrain/ncov/genbank_ingest:master?tab=info
+
+Connect any workspace variables to the wdl ingest workflow
+===========================================================
+  
+1. Navigate back to the **Workflow** tab, and click on the workflow imported to your workspace.
+2. Click on the radio button **Run workflow(s) with inputs defined by data table**.
+3. Under **Step 1**:
+
+  1. Select root entity type as **ncov_examples** from the drop down menu.
+
+4. Under **Step 2**:
+
+  1. Click **SELECT DATA**.
+  2. Select **Choose specific ncov_examples to process**.
```

Comment:
This shows on Terra as "Choose specific **ncov_exampless** to process". It looks weird, can it be renamed so there is only one plural `s`?

Reply:
Yes, would `ncov_example_set` work? 

That way, Terra will display "Choose specific **ncov_example_sets** to process". I'm open to other suggestions.

Reply:
I don't understand what a "set" is in Terra, but I'm not sure we want to be using the `_set` suffix. I noticed that most data tables created in our Development workspace are accompanied by another table with the same name suffixed with `_set`. This other table might be auto-created when data is selected in the process of running a workflow.

I just created a data table `victorlin_test_set` and noticed that it generated an additional column `victorlin_tests`. This doesn't happen without the `_set` suffix.

There's a doc page on [set tables](https://support.terra.bio/hc/en-us/articles/6660506445339-How-to-make-a-set-table-) which seems relevant.